CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 🚩 Deprecations 🚩
+
+- (Splunk) `Linux installer script`: Fluentd support has been deprecated and will be removed in a future release. ([#6264](https://github.com/signalfx/splunk-otel-collector/pull/6264))
```

Comment:
Perhaps we want to add a note about replacement, what should be used instead? Something along the lines of "please use native OTel receivers instead"? I'm not sure how to make it more specific at this time.

Reply:
Yes, I guess we can point to the logs collection config https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector/logs_config_linux.yaml

Reply:
I've added more specific replacement details, let me know what you all think 👍